funnel/views/email.py

```diff
@@ -139,6 +151,38 @@ def send_email_for_organization_admin_membership_revoked(
     )
 
 
+@signals.user_registered_for_project.connect
+def send_email_for_project_registration(project, user):
+    background_project_email.queue(
+        project_id=project.id, user_id=user.id, locale=get_locale()
+    )
+
+
+@rq.job('funnel')
+def background_project_email(project_id, user_id, locale):
+    project = Project.query.get(project_id)
+    user = User.query.get(user_id)
```

Comment:
Move these two inside the `with`. It doesn't matter here, but only because `__init__.py` has this line: `db.app = app`, telling the database engine to look for config in `app`. If we remove that line for any reason, these lines will fail. Moving inside `app_context` will resolve that.